class/defaults.yml

```diff
@@ -134,3 +158,10 @@ parameters:
           registry: quay.io
         master:
           labels: ${keycloak:labels}
+        volumePermissions:
+          enabled: ${keycloak:database:tls:enabled}
```

Comment:
Might not work in Openshift, I haven't tested...

It's apparently needed when using TLS:
https://github.com/bitnami/charts/tree/master/bitnami/postgresql#securing-traffic-using-tls